server/constants/messages.go

```diff
@@ -14,6 +14,7 @@ const (
 	TaskPreviewMessage   = "State: %s\nAssigned To: %s\nDescription: %s"
 	AlreadyLinkedProject = "This project is already linked."
 	NoProjectLinked      = "No project is linked, please link a project."
+	NoSubscriptionFound  = "No boards subscription exists for this channel"
```

Comment:
```suggestion
	NoSubscriptionFound  = "No board's subscription exists for this channel"
```

Reply:
I added this before but on Azure portal they are always writing `boards` also we can verify all these text messages from Mike later